wagn/lib/wagn/commands.rb

```diff
@@ -89,76 +79,30 @@ def find_spec_file filename, base_dir
       Dir.glob "#{p}/features"
     end.flatten
     require_args = "-r #{Wagn.gem_root}/features "
-    require_args += feature_paths.map { |path| "-r #{path}"}.join(' ')
+    require_args += feature_paths.map { |path| "-r #{path}" }.join(' ')
     feature_args = ARGV.empty? ? feature_paths.join(' ') : ARGV.join(' ')
-    unless system "RAILS_ROOT=. bundle exec cucumber #{require_args} #{feature_args} 2>&1"
+    unless system "RAILS_ROOT=. bundle exec cucumber " \
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.